routes/admin.js

```diff
@@ -69,8 +75,8 @@ function notLoggedIn(req, res, next) {
 
 // admin authentication function
 function isAdmin(req, res, next) {
-    if (req.user._id.toString() === process.env.ADMIN) {
+     if (req.user._id.toString() === process.env.ADMIN) {
```

Comment:
The "process.env.ADMIN" is still in the code and being validated on every admin route.